GitUI/CommandsDialogs/SettingsDialog/Pages/SortingSettingsPage.cs

```diff
@@ -0,0 +1,93 @@
+﻿using GitCommands;
+using GitCommands.Utils;
+using GitExtUtils.GitUI;
+using GitUIPluginInterfaces;
+using ResourceManager;
+
+namespace GitUI.CommandsDialogs.SettingsDialog.Pages
+{
+    public partial class SortingSettingsPage : SettingsPageWithHeader
+    {
+        private readonly TranslationString _revisionSortWarningTooltip = new("Sorting revisions may delay rendering of the revision graph.");
+        private readonly TranslationString _prioBranchNamesTooltip = new("Regex to prioritize branch names in the side panel and commit info.\n" +
+            "The branches matcing the pattern will be shown before the others.\n" +
+            "Separate the priorities with ';'.");
+        private readonly TranslationString _prioRemoteNamesTooltip = new("Regex to prioritize remote names in the side panel.\n" +
+            "The remotes matcing the pattern will be shown before the others.\n" +
```

Comment:
```suggestion
            "The remotes matcing the pattern will be shown before the others.\n" +
```
```suggestion
            "The remotes matching the pattern will be shown before the others.\n" +
```
Can we assign the tooltips in the designer?

Reply:
PictureBox like RevisionSortOrderHelp does not have ToolTip, they are assigned with ToolTip.SetToolTip() 
So not easily

Fixing the spellings